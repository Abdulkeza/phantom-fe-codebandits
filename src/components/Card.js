import React from 'react';

const Card = ({ image, alt, title, text, styles }) => {
  return (
    <div
      className={`${styles} border w-52 h-52 flex flex-col justify-center items-center rounded-lg shadow-md`}
    >
      <img src={image} alt={alt} className="w-12 mt-4" />
      <h3 className="py-4 font-extrabold text-sm">{title}</h3>
      <p className="text-center text-sm my-2">{text}</p>
    </div>
  );
};

export default Card;
