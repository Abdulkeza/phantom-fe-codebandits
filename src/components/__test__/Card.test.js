import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../Card.js';

describe('Card', () => {
  it('should render Card', () => {
    const elem = renderer.create(<Card />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
