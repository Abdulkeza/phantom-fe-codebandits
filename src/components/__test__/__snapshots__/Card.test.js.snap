// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should render Card 1`] = `
<div
  className="undefined border w-52 h-52 px-4 text-center flex flex-col justify-center items-center rounded-lg shadow-md hover:bg-hover2"
>
  <img
    className="w-12 mt-4"
  />
  <h3
    className="font-raley mt-2 font-extrabold text-sm"
  />
  <p
    className="font-sans text-center text-sm my-2"
  />
</div>
`;
